content/docs/user-guide/appendix/available-ebus.md

```diff
@@ -57,7 +44,7 @@ You can use the `SessionEventBus` to detect when a member joins or leaves a netw
 | Methods | AddImpulse, GetMass, SetMass, GetVelocity, SetVelocity, etc. |
 | File | PhysicsComponentBus.h |
 
-The `PhysicsComponentRequestBus` contains useful methods for getting or setting the physical characteristics of objects like mass, density, velocity, and water damping. For an example of using a pointer directly to the underlying handler for better access to functions such as `GetVelocity`, see [Direct Access to EBus Handlers](/docs/userguide/programming/ebus/handlers-direct-access.md).
+The `PhysicsComponentRequestBus` contains useful methods for getting or setting the physical characteristics of objects like mass, density, velocity, and water damping. 
```

Comment:
Was PhysicsComponentBus.h part of the legacy physics system? I can't locate it. This section may need to be removed.

Reply:
Looks correct since it's a `LmbrCentral` bus. It may have been renamed but I can pull it for now.

Reply:
Went ahead and switched the whole article to `draft` because it's very out of date - even the handlers which still exist (like input handlers) are now located in other source code and may have different methods/messages on them.